### PR TITLE
code review comments

### PR DESCRIPTION
**Pull request by mcassels**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -183,7 +183,7 @@ fn write_polygon<T: NumTy>(buffer: &mut Vec<u8>, p: &Polygon<T>) {
     let num_rings = (interiors.len() + 1) as u32;
     write_u32(buffer, num_rings);
     write_ring(buffer, exterior);
-
+    // is there somewhere in the spec that says that the exterior is always the first ring? Or is that an arbitrary choice?
```
Comment (maxburke) [inline src/lib.rs:186]: I can't find it in any specs but it's typically the case that the first ring is the exterior.

File: src/lib.rs [hunk 2]
```diff
@@ -153,7 +153,7 @@ fn write_ring<T: NumTy>(buffer: &mut Vec<u8>, ring: &LineString<T>) {
 
 #[inline]
 fn write_point<T: NumTy>(buffer: &mut Vec<u8>, p: &Point<T>) {
-    buffer.push(1);
+    buffer.push(1); // minor: we could use `Endian::Little` instead of `1` here (and in every write function) to be more explicit
```
Comment (maxburke) [inline src/lib.rs:156]: Done

File: src/lib.rs [hunk 3]
```diff
@@ -249,7 +249,7 @@ impl<T: NumTy> TryFrom<Geometry<T>> for Wkb {
 }
 
 fn read_coordinate<T: NumTy>(reader: &mut LittleEndianReader) -> Result<Coordinate<T>, WkbError> {
-    let x: T = reader.read_f64()?.try_into().unwrap();
+    let x: T = reader.read_f64()?.try_into().unwrap(); // Instead of unwrapping could we return an error here?
```
Comment (maxburke) [inline src/lib.rs:252]: I think I can make these just `.into()`

**Comment by mcassels** [comment]
comments addressed; closing

**Pull request closed without merging**